dev_tools/notebooks/isolated_notebook_test.py

```diff
@@ -36,7 +36,12 @@
 # these notebooks rely on features that are not released yet
 # after every release we should raise a PR and empty out this list
 # note that these notebooks are still tested in dev_tools/notebook_test.py
-NOTEBOOKS_DEPENDING_ON_UNRELEASED_FEATURES = []
+NOTEBOOKS_DEPENDING_ON_UNRELEASED_FEATURES = [
```

Comment:
is there something in the release procedure to check that that these get removed after a release?

Reply:
it's all manual, but the instructions are there in release.md